UI/NpcList/NPCRow.gd

```diff
@@ -0,0 +1,54 @@
+extends PanelContainer
+
+onready var _npcNameLabel = $HBoxContainer/Name
+onready var _npcGenderLabel = $HBoxContainer/Gender
+onready var _npcPersonalityLabel = $HBoxContainer/Personality
+onready var _amountOfChildrenLabel = $HBoxContainer/Children
+onready var meetButton = $HBoxContainer/CenterContainer2/Meet
+onready var forgetButton = $HBoxContainer/CenterContainer/Forget
+onready var showNpc = $ShowNPC
+var _npcID: String
+
+
+func setName(name: String):
+	_npcNameLabel.text = name
+
+
+func setGender(gender: String):
+	_npcGenderLabel.text = gender
+
+
+func setPersonality(personality: String):
+	_npcPersonalityLabel.text = personality
+
+
+func setChildren(children):
+	_amountOfChildrenLabel.text = String(children)
+
+
+func setNpcID(ID: String):
+	if(_npcID == null):
+		push_error("Exception: Attempt to set null NPC in the NPC list")
+	else:
+		_npcID = ID
+
+
+func getNpcID():
+	if(_npcID != null):
```

Comment:
since you declared it as `_npcID: String` it will never be null ^^ (I think at least x3). You can still check it against an empty string